contribuer/public/admin/config.yml

```diff
@@ -237,7 +237,7 @@ fields:
     label: Courte description
     name: description
     widget: description
-  field_conditions_non_prise_en_compte: &field_conditions_non_prise_en_compte
+  field_unmodeled_conditions: &field_unmodeled_conditions
```

Comment:
J'avais un doute sur le sens de ``non modélisées`` pour ``non prises en compte`` mais je manque peut-être de contexte :)
En alternative au cas où : ``not included``, ``not taken into account``, ``not considered``